File: 4_CountingDigits/4.py
def contarDigitos(n):
    cont = 0
    while n != 0:
        n = n / 10;
        cont = cont + 1;
    return cont

n = int(input(">> n: "))
print(str(n) + " tiene " + str(contarDigitos(n)) + " digito(s).")

File: 4_CountingDigits/4.rb
def contarDigitos(n)
    cont = 0
    while n != 0
        n = n / 10
        cont += 1
    end
    return cont
end

print ">> n: "
n = gets.chomp.to_i
puts "#{n} tiene #{contarDigitos(n)} dígito(s)"
